drivers/adc/ad7779/ad7779.h

```diff
@@ -241,7 +242,7 @@ typedef enum {
 
 typedef struct {
 	/* SPI */
-	struct no_os_spi_desc		*spi_desc;
```

Comment:
do not remove the explicit declaration (`struct`), we removed the typedefs with https://github.com/analogdevicesinc/no-OS/commit/b21f3caad8f1ce1296e51ade11259b082bf3c361

Reply:
Reverted this- struct has been added back now